### PR TITLE
Initial BLED112 implementation

### PR DESCRIPTION
**Pull request by roywant**
Please review initial code base. Note this code is functional and currently passes all the validator tests.

### CODE REVIEW

**Comment by roywant** [comment]
@schilit can you take a look?

File: beacons/BLED112/cdc.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (schilit) [inline beacons/BLED112/cdc.xml]: Try running through http://www.webtoolkitonline.com/xml-formatter.html

**Comment by roywant** [comment]
@savolmi  please can you provide review comments for this BLED112 Uribeacon implementation.

--Roy

File: beacons/BLED112/uribeacon.bgs [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+   call hardware_set_txpower(radiotxpowerlevels(2:1))  # Set MEDIUM Power in Radio
+
+   # SET SCAN RESPONSE PARAMS
+
+   # Flags = LE General Discovery, single mode device (02 01 06)
+   advConfSrData(0:1)  = 2   # Length
+   advConfSrData(1:1)  = $01 # AD Flags
+   advConfSrData(2:1)  = $06 # 06=connectable 04=non-connectable
+
+   # Name= "Config Uribeacon BG"
```
Comment (schilit) [inline beacons/BLED112/uribeacon.bgs]: probably want to put this back to "Config UriBeacon" without the cryptic "BG"

**Comment by valterforesto** [comment]
Hi @roywant Roy,
I have reviewed and tested yours suggested BLE112 implementation in BGScript at Pull
Request 116 #166 that works fine.
Valter

**Pull request merged**